llvm/lib/Target/RISCV/RISCVRegisterInfo.td

```diff
@@ -416,12 +416,11 @@ class VRegList<list<dag> LIn, int start, int nf, int lmul, bit isV0> {
 }
 
 // Vector registers
-foreach Index = 0-31 in {
+foreach Index = !range(0, 32, 1) in {
```

Comment:
Why is this better?

Reply:
It's not better, just to align other `foreach`s below. It may be more uniformed from the style perspertive.

Reply:
Thanks! `IndexSet` is rewritten.